tests/ha/test_data_pod_failure.py

```diff
@@ -2278,6 +2278,7 @@ def test_copy_object_unsafe_pod_shutdown(self):
     @pytest.mark.ha
     @pytest.mark.lc
     @pytest.mark.tags("TEST-32458")
+    @pytest.mark.skip(reason="VM issue in after Restart(LRL-3413). Need to be tested on HW.")
```

Comment:
Restart node related tests need to be skipped because of [CORTX-32933](https://jts.seagate.com/browse/CORTX-32933).
Above LRL is closed.